sdk/appconfiguration/Azure.ApplicationModel.Configuration/tests/ConfigurationLiveTests.cs

```diff
@@ -12,19 +12,25 @@
 
 namespace Azure.ApplicationModel.Configuration.Tests
 {
-    [Category("Live")]
-    public class ConfigurationLiveTests: ClientTestBase
+    public class ConfigurationLiveTests: RecordedTestBase
     {
-        public ConfigurationLiveTests(bool isAsync) : base(isAsync) { }
+        public ConfigurationLiveTests(bool isAsync) : base(isAsync)
+        {
+            Sanitizer = new ConfigurationRecordedTestSanitizer();
+            Matcher = new RecordMatcher(Sanitizer);
+        }
 
         private string GenerateKeyId(string prefix = null)
         {
-            return prefix + Guid.NewGuid().ToString("N");
+            return prefix + Recording.GenerateId();
```

Comment:
I feel a little conflicted about the name `Recording` instead of something like `Testing` because most of these uses aren't specific to recording, just have behavior that gets changed when we are recording/playing back.  I don't feel strongly enough to change it if no one else has an issue.